Scatternest/Scatternest.cs

```diff
@@ -153,7 +153,8 @@ private void OnExportCompleted(RandoController rc)
 
         public void HookItemSync()
         {
-            RandoController.OnExportCompleted += SelectStartIndex;
+            RandoController.OnExportCompleted += RandoControllerSelectStartIndex;
+            if (ModHooks.GetMod("ICDL Mod") is Mod) On.UIManager.StartNewGame += ICDLSelectStartIndex;
```

Comment:
Could this be made to work with one of the ItemChanger start new game hooks? Most likely the [BeforeStartNewGame](https://github.com/homothetyhk/HollowKnight.ItemChanger/blob/master/ItemChanger/Events.cs#L35) hook

Reply:
Seems to work!  Done.